src/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,59 @@
+import cn from 'classnames';
+import { memo } from 'react';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todo: Todo,
+  onDeleteTodo: (id: number) => void,
+  hasLoadingStatus: boolean,
+}
+
+export const TodoInfo: React.FC<Props> = memo(({
+  todo,
+  onDeleteTodo,
+  hasLoadingStatus,
+}) => {
+  const { title, completed, id } = todo;
+
+  return (
+    <div
+      data-cy="Todo"
+      className={cn('todo', {
+        completed: completed === true,
+      })}
+    >
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={completed}
+        />
+      </label>
+
+      <span data-cy="TodoTitle" className="todo__title">
+        {title}
+      </span>
+
+      <button
+        type="button"
+        className="todo__remove"
+        data-cy="TodoDelete"
+        value={id}
+        onClick={(e) => onDeleteTodo(+e.currentTarget.value)}
+      >
+        ×
+      </button>
+
+      <div
+        data-cy="TodoLoader"
+        className={cn('modal overlay', {
+          'is-active': hasLoadingStatus === true,
```

Comment:
```suggestion
          'is-active': loadingTodosIds.includes(id), // or pass directly this value to todo info component
```
Now when I delete one todo I see loaders on all todos instead. Loader should be shown only for todos that are actually loading, you can check if current todo id is in the loadingTodosIds array